SignalServiceKit/src/Util/OWSFileSystem.h

```diff
@@ -8,7 +8,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (instancetype)init NS_UNAVAILABLE;
 
-+ (void)protectFolderAtPath:(NSString *)path;
+// TODO: We shouldn't ignore the return value of thie method.
++ (BOOL)protectFolderAtPath:(NSString *)path;
```

Comment:
Want to add `__attribute__((warn_unused_result))`?

Reply:
Done, and I've added a card to do an audit.